modules/ROOT/pages/salesforce-releases-migrations.adoc

```diff
@@ -0,0 +1,24 @@
+= Salesforce Releases and Migrations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+The Salesforce organization dedicated to your API community follows the standard https://trailhead.salesforce.com/en/content/learn/modules/sf_releases/sf_releases_start[Salesforce release process]. This automatically upgrades your organization to the latest Salesforce release version three times each year. As an organization administrator, you'll receive email notifications about the changes at least three months in advance, including any required steps for you to follow, guidance, and other resources.
```

Comment:
present tense
```suggestion
The Salesforce organization dedicated to your API community follows the standard https://trailhead.salesforce.com/en/content/learn/modules/sf_releases/sf_releases_start[Salesforce release process]. This automatically upgrades your organization to the latest Salesforce release version three times each year. As an organization administrator, you receive email notifications about the changes at least three months in advance, including any required steps for you to follow, guidance, and other resources.
```
"steps, guidance, and resources" sounds odd to me: you? Also, "other resources" is vague.